src/Tribe/Assets.php

```diff
@@ -705,7 +710,7 @@ public function register( $origin, $slug, $file, $deps = [], $action = null, $ar
 
 		// Clean these
 		$asset->priority  = absint( $asset->priority );
-		$asset->in_footer = (bool) $asset->in_footer;
+		$asset->in_footer = is_array( $asset->in_footer ) ? $asset->in_footer : (bool) $asset->in_footer; // Since WordPress 6.3, this parameter accepts an array argument.
```

Comment:
Lets not change how `in_footer` works, when we see the new version of WordPress just modify how we use the values.

Reply:
@bordoni we have to change something here or the `array` will be cast to a `bool` everytime. Remove the casting altogether? Not sure whether this is here for a particular situation and if it will be safe to remove the casting.

Reply:
https://github.com/the-events-calendar/tribe-common/pull/2054
Don't use `in_footer` - use existing `async`/`defer` params

Reply:
What I meant was, dont use the `in_footer` from Tribe Assets, use the already existing `async`/`defer` we have plus the `in_footer` value to compose the array for WordPress.